### ISSUE TITLE
Feature Request: Guzzle 6.0 Support

### ISSUE THREAD

**Issue by collinstowell** [fillup/walmart-partner-api-sdk-php#43]
I'm not sure if this project is still being actively maintained, but it would be nice if support for Guzzle 6.0 were added. If not, I can fork it and support myself.

**Comment by fillup** [comment]
I agree, it would be nice now that Guzzle 6 supposedly supports the guzzle-services stuff. I haven't tried it out yet but I've seen the repos update and it should work. I just don't know what size effort it is and how much has changed in service definitions, etc. I'm open to PRs if you want to give it a shot and submit that way. As for is this project still maintained I'd say sort of. I'll try to catch up on bugs and feature requests but its not my main priority anymore so it can be hard to get to it.

**Comment by adipaulo** [comment]
Fillup, thanks for this great contribution ...
do you have any update about this  feature request (guzzle ~6) ?
thanks again

**Comment by adipaulo** [comment]
any update for Guzzle 6* ?
